flusher.go

```diff
@@ -144,6 +147,10 @@ func (s *Server) flushSink(
 				continue metricLoop
 			}
 			if sink.maxNameLength != 0 && len(metric.Name) > sink.maxNameLength {
+				s.Statsd.Count("dropped_metrics", 1, []string{
```

Comment:
Overall LGTM, one question: is `flushed_metrics` with `status != flushed` redundant with this? Do we want to combine them into a single metric that counts everything?

Reply:
This metric is more descriptive since it also contains the metric name thats actually being dropped, I'd treat this as a more descriptive "lower level" metric that we can use for fine grained analysis where flushed_metric is higher level and can be used for cheaper overall dashboarding/alerting.